### PR TITLE
OGM-1022 Update documentation around MongoDB CLI syntax

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/OGM-1022

 Add examples for $orderby, .limit() alternative, update, remove and insert.

### CODE REVIEW

File: documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc [hunk 1]
```diff
@@ -3254,32 +3256,68 @@ No cursor operations such as `sort()` are supported.
...
 
+You can limit the results of a query using the `setMaxResults(...)` method.
+
 JSON parameters passed via the CLI syntax must be specified using the
 http://docs.mongodb.org/manual/reference/mongodb-extended-json/[strict mode].
 Specifically, keys need to be given within quotes; the only relaxation of this is that single quotes
 may be used when specifying attribute names/values to facilitate embedding queries within
 Java strings.
 
+Note that results of projections are returned as retrieved from the MongoDB driver at the moment and
```
Comment (gunnarmorling) [inline documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc]: Is there a JIRA issue for this already? Could you create one if not, and reference it here?

Comment (gunnarmorling) [inline documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc]: Nevermind, created [HV-1031](https://hibernate.atlassian.net/browse/OGM-1031).

**Comment by gunnarmorling** [comment]
Rebased and applied. Added a reference to OGM-1033 for projection value conversion. Thanks, @DavideD!

File: documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc (hunk 1 above)
Comment (DavideD) [inline documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc]: ok, thanks

**Comment by DavideD** [comment]
> Rebased and applied. Added a reference to OGM-1033 for projection value conversion. Thanks, @DavideD!

It's [OGM-1031](https://hibernate.atlassian.net/browse/OGM-1031)

**Comment by gunnarmorling** [comment]
Ah, yes. Fixed.

**Pull request closed without merging**